main.py

```diff
@@ -62,4 +62,29 @@ async def sync(ctx: commands.context.Context):
         print(e)
 
 
+@bot.command()
+async def clear(ctx: commands.context.Context):
+    bot.tree.clear_commands(guild=ctx.guild)
+
+    try:
+        await bot.tree.sync()
+        await ctx.send("Commands cleared.")
+    except Exception as e:
+        print(e)
+
+
+@bot.command()
+@commands.has_role(EXEC_ID)
+async def sethintchannel(ctx: commands.context.Context):
+    channel = ctx.channel
+
+    # set new channel id in environment variable
+    os.environ["ADMIN_CHANNEL_ID"] = str(channel.id)
+
+    # change the config variable
+    config["ADMIN_CHANNEL_ID"] = channel.id
+
```

Comment:
oh.. if you want to be able to modify the admin channel in code, this is not possible :c you'll need to make a table on postgres and store it. overwriting the environment variable only persist for the current container.

Reply:
Yep, I know. In reality this command won't be needed since I don't foresee the hint channel needing to change.